### ISSUE TITLE
Startup Beeps annoyance

### ISSUE THREAD

**Issue by DaLiV** [bigtreetech/BIGTREETECH-TFT35-V3.0#158]
Buzzer tone is annoying on power on
would be nice to bypass alerts till power-on complete, or for first some "boot-time" seconds

**Comment by radek8** [comment]
Are you using current FW?
last build is 18 days old...

**Comment by DaLiV** [comment]
yes - latest Bigtreetech_GD_TFT35_V3.0 V27.x Oct 7 2022 in 120Mhz
marlin mode - no any sounds
but in serial - that gives approx 24 beeps 
TFT connected to the board "octopus pro" (last is not connected to any sensors at present)
